Bio/motifs/minimal.py

```diff
@@ -0,0 +1,162 @@
+# By Ariel Aptekmann 2017
+# Adapted from Bio.motifs 2008 by Bartek Wilczynski
+# Adapted from  Bio.MEME.Parser by Jason A. Hackney.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+from __future__ import print_function
+
+from Bio.Alphabet import IUPAC
+from Bio import Seq
+from Bio import motifs
+import math
+
+def read(handle):
+    """Parses the text output of the MEME program into a meme.Record object.
+    Example:
+    >>> from Bio.motifs import meme
```

Comment:
I think you will need a blank line between "Example:" and the doctest, this will probably be triggering an RST validation failure in our TravisCI style checks.